WalletWasabi.Fluent/ViewModels/Wallets/LoadingViewModel.cs

```diff
@@ -50,6 +50,8 @@ public LoadingViewModel(Wallet wallet)
 
 	private uint RemainingFiltersToDownload => (uint)Services.BitcoinStore.SmartHeaderChain.HashesLeft;
 
+	private bool IsNewlyGeneratedWallet => _filtersToProcessCount <= 1; // 1 -> the wallet was created approximately 10 minutes ago
```

Comment:
What does this do?

What if I create a wallet, then wait for a day, before loading the wallet for first time?

It should still skip filter scan.
The keys are new, the wallet never loaded, there can not be any coins on there.

Reply:
The only information we can decide on is the height of the wallet.